static/js/Promotions.jsx

```diff
@@ -151,6 +152,32 @@ function trackSidebarAdClick(ad) {
     adType: "sidebar",
   });
 }
+const cookie = Sefaria._inBrowser ? $.cookie : Sefaria.util.cookie;
```

Comment:
I should have mentioned this last review, use localStorage as that is our current convention for these kind of things https://github.com/Sefaria/Sefaria-Project/blob/ea38194f629487cf97458b75346ae5b84844f8e0/static/js/sefaria/sefaria.js#L2491